external-crates/move/crates/move-cli/src/sandbox/utils/on_disk_state_view.rs

```diff
@@ -112,6 +112,23 @@ impl OnDiskStateView {
         Self::get_bytes(&self.get_module_path(module_id))
     }
 
+    /// Read the package bytes stored on-disk at `addr`
+    fn get_package_bytes(&self, address: &AccountAddress) -> Result<Option<Vec<Vec<u8>>>> {
```

Comment:
This return type seems janky -- shouldn't we give an error if the address path is invalid?

Reply:
This is in keeping with the other APIs in here/in general (e.g., `get_module_bytes)` with storage-related APIs, so I think this type makes sense to me at least? 

If it's useful -- this is used for the `ModuleResolver` trait below in this module, and this trait is also over DBs where you do want to signify "error fetching" vs "no error but not found"